spec/features/visitors/i18n_spec.rb

```diff
@@ -52,15 +52,17 @@
 
     it 'allows user to manually toggle language from dropdown menu', js: true do
       visit root_path
-      within(:css, '.i18n-desktop-dropdown', visible: false) do
-        find_link(t('i18n.locale.es'), visible: false).trigger('click')
+      within(:css, '.i18n-desktop-toggle') do
+        click_link t('i18n.language', locale: 'en')
+        click_link t('i18n.locale.es')
```

Comment:
This better simulates what a user will click on, as opposed to trying to click on invisible elements.